ruby_event_store_rom_sql/lib/ruby_event_store/index_violation_detector.rb

```diff
@@ -0,0 +1,24 @@
+module RubyEventStore
+  class IndexViolationDetector
```

Comment:
Nice!

Reply:
@pawelpacana this is copied from AR ... thoughts on creating a common SQL library or something?

Reply:
would make sense to have it in AR as well after merge